trino/__init__.py

```diff
@@ -20,4 +20,6 @@
 from . import exceptions
 from . import logging
 
+__all__ = ['auth', 'dbapi', 'client', 'constants', 'exceptions', 'logging']
```

Comment:
I'm not sure if we consider all of these to be public API.

Most notably `client` isn't supposed to be public since it's an implementation detail of the driver. People can use it but it's not expected to be stable (which is what `__all__` indicates by convention.

`auth`, `dbapi`, `constants` and `exceptions` might be considered public.

Reply:
`__all__` is updated here to indicate to flake8 that the unused imports are part of a public API. Otherwise, we might want to remove `client` and `logging` from `__init__.py`. What do you think?

Reply:
Good point. Let's leave it out of this PR - I'll follow up on this in https://github.com/trinodb/trino-python-client/issues/43